### PR TITLE
Added support for the worklog endpoints

### PR DESCRIPTION
**Pull request by Brian-Chow**
I added support for the worklog REST endpoints from Atlassian.  It would be appreciated if they were merged into the main branch and npm updated.

### CODE REVIEW

File: api/worklog.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+module.exports = WorklogClient;
+
+/**
+ * Used to access Jira REST endpoints in '/rest/api/2/jql/autocompletedata'
```
Comment (floralvikings) [inline api/worklog.js]: You mind updating these docs?  They indicate that they're pointing to the JQL `autocompletedata` endpoint.

**Comment by floralvikings** [comment]
Left a comment about the docs; other than that, looks great!  I'll merge and publish as soon as the docs are corrected.

**Comment by Brian-Chow** [comment]
Updated docs to reflect correct url.  Sorry about that.

**Comment by floralvikings** [comment]
Awesome, will merge and publish

**Comment by floralvikings** [comment]
Published as version 2.1.0

Thanks!

**Comment by Brian-Chow** [comment]
Thank you very much!

**Pull request merged**